modules/workbench/modules/reporting/assets/schemas/verified_snapshot.json

```diff
@@ -0,0 +1,12 @@
+ [
+   {
+     "description": "Time snapshot was taken, in Epoch milliseconds. Same across all rows and all tables in the snapshot, and uniquely defines a particular snapshot.",
+     "name": "snapshot_timestamp",
+     "type": "INTEGER"
+   },
+  {
```

Comment:
I suggest just omitted this second column, since the table is already called `verified_snapshot`, presence in the table should be sufficient